src/remote-checkout/remote-checkout-action-creator.ts

```diff
@@ -91,6 +91,21 @@ export default class RemoteCheckoutActionCreator {
         });
     }
 
+    forgetCheckout(methodId: string, options?: RequestOptions): Observable<Action> {
+        return Observable.create((observer: Observer<Action>) => {
+            observer.next(createAction(RemoteCheckoutActionType.SignOutRemoteCustomerRequested, undefined, { methodId }));
```

Comment:
Can we please create new action types for this action and not reuse the ones from `SignOut`, since they are separate actions.

Reply:
Also think you may need to reload the checkout as part of this, since there might be some information related to session that is not cleared for the shopper.

Reply:
I don't think this needs to be reloaded because forgetCheckout deletes all the checkout information from the backend and in this case there is no information that needs to be loaded again in the frontend.

Reply:
@BC-PabloRivera `forgetCheckout` only removes certain session data. It doesn't wipe out all the checkout information. Most of the fields of checkout won't change after removing those session data, except for `payments` field. If you don't reload it, the browser client would be referencing stale data.

Reply:
Ok. When you talk about reloading the checkout, is something like `checkoutActionCreator.loadCurrentCheckout()` what you expect or do you have something different in mind?

Reply:
Sorry for the late reply, you should be able to call that in the action itself.